src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -2321,6 +2325,8 @@ def __init__(self, config, *inputs, **kwargs):
         checkpoint=_CHECKPOINT_FOR_DOC,
         output_type=TFLongformerSequenceClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_output="LABEL_0",
+        expected_loss=0.58,
```

Comment:
I think the checkpoint `_CHECKPOINT_FOR_DOC`, which is `allenai/longformer-base-4096` doesn't contain sequence classification head, and the head is randomly initialized. The result would be random, no?

Reply:
This is correct!